tests/fpgadataflow/test_convert_to_hls_conv_layer.py

```diff
@@ -156,10 +163,15 @@ def test_convert_to_hls_conv_layer(conv_config, depthwise, exec_mode):
     x = gen_finn_dt_tensor(idt, input_shape)
     inp_dict = {model.graph.input[0].name: x}
     assert oxe.compare_execution(model, new_model, inp_dict)
+
+    if use_rtl_swg:
+        downsampler_op_type = "ConvolutionInputGenerator_rtl"
+    else:
+        downsampler_op_type = "DownSampler"
     if kernel_size == 1 and stride > 1 and pad == 0:
-        assert new_model.graph.node[1].op_type == "DownSampler"
+        assert new_model.graph.node[1].op_type == downsampler_op_type
```

Comment:
maybe I missed something here but there is a skip condition above:

```
    if use_rtl_swg and kernel_size == 1:
        pytest.skip("1x1 kernel not supported by current RTL SWG")
```

if the 1x1 kernel is not support by the RTL SWG and its testcase will be skipped, does this piece of code ever get triggered?

Reply:
You're right, this is a remnant of the "parallel mode" that supported this case. I removed this piece of code for now.